.github/workflows/test.yml

```diff
@@ -16,7 +16,18 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  test-scripts:
```

Comment:
this will be better in the lint.yml workflow, so we don't pollute our main workflow with skipped jobs as in https://github.com/kernel-patches/bpf/actions/runs/6582407583